packages/ui/src/components/va-select/VaSelect.vue

```diff
@@ -674,6 +671,12 @@ export default defineComponent({
     border-top-left-radius: 0;
     box-shadow: var(--va-select-box-shadow);
     padding: 0;
+
+    .va-input {
+      .va-input-wrapper__content {
+        flex-grow: 1;
```

Comment:
Duplicated thing 2.

I think that style should be inside VaInput, instead of VaSelect.

Reply:
Replaced duplicated